scripts/data_contamination/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3
+
+RUN mkdir -p /script/src
+
+WORKDIR /script/src
+
+COPY . .
+
+RUN pip install -r ./requirements.txt
+
+CMD python compute_contamination_metrics.py --input-data $INPUT_PATH --scenario-data $SCENARIO_DATA --output-stats $OUTPUT_STATS --input-format $INPUT_FORMAT
```

Comment:
`ENTRYPOINT python compute_contamination_metrics.py`

Then you can just do `docker run contamination_script:latest --input-data input_path` etc. and it should just work. Also `--help` would work as expected.

Reply:
This doesn't work for me. I get:
```
(base) ~/research/helm/scripts/data_contamination (andyz/docker_file ✗) docker run contamination_script:latest  --input-data="input.json" --scenario-data="scenario_data" --output-stats="output_stats" --input-format="the_pile"

usage: compute_contamination_metrics.py [-h] --input-data INPUT_DATA
                                        --scenario-data SCENARIO_DATA
                                        --output-stats OUTPUT_STATS
                                        --input-format INPUT_FORMAT
                                        [--tags [TAGS [TAGS ...]]]
                                        [--normalization NORMALIZATION]
                                        [--output-ngrams OUTPUT_NGRAMS]
                                        [--max-output-ngrams MAX_OUTPUT_NGRAMS]
compute_contamination_metrics.py: error: the following arguments are required: --input-data, --scenario-data, --output-stats, --input-format
```
Would you mind checking out the branch to verify that the command works as expected for you?